win32ss/user/ntuser/message.c

```diff
@@ -1867,8 +1867,8 @@ co_IntSendMessageWithCallBack( HWND hWnd,
 
     if(!(Message = AllocateUserMessage(FALSE)))
     {
-        ERR("MsqSendMessage(): Not enough memory to allocate a message");
-        RETURN( FALSE);
+        ERR("MsqSendMessage(): Not enough memory to allocate a message\n");
+        RETURN(FALSE);
```

Comment:
Perhaps I merged this too fast, are you sure this is correct change?

Reply:
Ok, looking at this macro, it seems correct: https://git.reactos.org/?p=reactos.git;a=blob;f=win32ss/user/ntuser/ntuser.h;hb=9f428f5522c9a6a4ec8cf19f7f0c543c944b9483#l4

Reply:
yes, i understand that the trailing space is non-functionnal and the "\n" is a must in ERR macro.